
int16 sender_ID
int16 message_Type
int16 protocol_Type
int16 service_Level
int16 packet_ID

DTAG_Message DTAG_Message
DTAP_Message DTAP_Message
GBS_Message  GBS_Message
SEBS_Message SEBS_Message